webapp/src/Controller/Jury/ProblemController.php

```diff
@@ -115,11 +115,11 @@ public function indexAction(): Response
             }
 
             // Create action links
-            if ($p->getProblemtextType()) {
+            if ($p->getProblemstatementType()) {
                 $problemactions[] = [
-                    'icon' => 'file-' . $p->getProblemtextType(),
-                    'title' => 'view all problem statements of the contest',
-                    'link' => $this->generateUrl('jury_problem_text', [
+                    'icon' => 'file-' . $p->getProblemstatementType(),
+                    'title' => 'view problem statement',
```

Comment:
Why did the singular change? Apparantly it was always only about 1 problem text.

Reply:
Im my previous PR I made the mistake of typing the wrong text, and I fixed that here.